packages/help-center/src/hooks/use-zendesk-messaging.ts

```diff
@@ -50,6 +50,7 @@ export default function useZendeskMessaging(
 	}, [ setMessagingScriptLoaded, enabled, zendeskKey, isMessagingScriptLoaded ] );
 
 	if (
+		document &&
```

Comment:
This is unrelated to my changes, but this fixes an error triggered when loading the `Plugins` [page](http://calypso.localhost:3000/plugins):

```javascript
ERROR calypso: document is not defined (reqId=cfe59271-5514-4a6c-bb65-d9e7643eaa5c, url=/plugins, env=development, userAgent="Chrome 121", path
=/plugins)
    ReferenceError: document is not defined
        at useZendeskMessaging (/var/sources/build/webpack:/wp-calypso/packages/help-center/src/hooks/use-zendesk-messaging.ts:52:2)
        at usePresalesChat (/var/sources/build/webpack:/wp-calypso/client/lib/presales-chat/index.ts:70:38)
        at PluginsBrowser (/var/sources/build/webpack:/wp-calypso/client/my-sites/plugins/plugins-browser/index.jsx:101:2)
        at renderWithHooks (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:5662:16)
        at renderIndeterminateComponent (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:5735:15)
        at renderElement (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:5950:7)
        at renderNodeDestructiveImpl (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:6108:11)
        at renderNodeDestructive (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:6080:14)
        at renderNode (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:6263:12)
        at renderHostElement (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:5646:3)
        at renderElement (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:5956:5)
        at renderNodeDestructiveImpl (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:6108:11)
        at renderNodeDestructive (/var/sources/build/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js:6080:14)

```

Reply:
Were we generating that on every page load? 😓